models/uniswap/ethereum/uniswap_v3_ethereum_proposals.sql

```diff
@@ -0,0 +1,71 @@
+{{ config(
+    schema = 'uniswap_v3_ethereum',
+    alias = 'proposals',
+    materialized = 'incremental',
```

Comment:
would it make sense to partition by ```block_date``` like other models? i.e. will downstream dashboards filter on or group by date? it would help performance in that case. if not, all good

Reply:
same concept to other model

Reply:
Yeah I thought about that but tables are really small (e.g., proposals is like 25 rows) so I didn't think it would make a difference, will still add it in case it does when a lot of DAOs are added in the future

Reply:
Done